client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -139,11 +198,18 @@ const EditPage = createClass({
 	},
 
 	trySave : function(){
-		if(!this.debounceSave) this.debounceSave = _.debounce(this.save, SAVE_TIMEOUT);
-		if(this.hasChanges()){
-			this.debounceSave();
-		} else {
-			this.debounceSave.cancel();
+		if(this.editTypeIsNew()) {
```

Comment:
`trySave()` is meant just to be a check so we don't save too frequently (we don't want to save with *every* character the user types or we overload the servers.) but instead limits to every 3 seconds. All the actual "save" logic needs to go in the `save()` function, but I think the "debounce" stuff can apply to both New and Edit pages.

Reply:
I'm just experimenting with a change now, instead of pushing the `/new` data to localStorage immediately on any change, it instead checks for New/Edit when `debounceSave` is triggered; if new it pushes the data to localStorage and cancels the debounce. If the type is Edit, it pushes the data through the normal `this.save()` function.

Reply:
That sounds reasonable to me.